Neos.Flow/Tests/Unit/Mvc/ActionRequestTest.php

```diff
@@ -342,8 +342,8 @@ public function getControllerSubpackageKeyReturnsTheUnknownCasesPackageKeyIfNoCo
     public function invalidControllerNames()
     {
         return [
-            [42],
-            [false],
+            //[42],
```

Comment:
@kitsunet should we add an explicit test for the controller (and below action) name not being a "numeric" inside the setter to still pass these tests? I mean, a "42Controller" is an invalid class name anyway.